### DISCUSSION TITLE
Can we upload react project which is calling MSSQL express database to Cloud for free?

### DISCUSSION THREAD

**Discussion by hb6777** [community/community#32480] [New to GitHub] [answered]
Hi, I am very new to GitHub. We have React projects which uses MSSQL express as backend. We are wondering if we can upload all projects to  GitHub Cloud for free? So others can access to our websites? Thank you very much.

**Comment by strong-728** [comment, marked as answer]
yes, you can. you can make the github repositories and push your project as a privates.
And then you can sync your github repo with the amplify, vercel and so on.
Good luck.
